### DISCUSSION TITLE
Support for using Turso and sqlite

### DISCUSSION THREAD

**Discussion by emilpriver** [umami-software/umami#2582] [General]
### Describe the feature or enhancement

Seems as Prisma is supporting SQLite and soon Turso(early access) and this would be really handy to use with Prisma.

https://www.prisma.io/docs/orm/overview/databases/turso

Is there anything on the roadmap for Turso and SQLite?

**Comment by Maxime-J** [comment]
The only db related thing I'm aware of on the roadmap is the ClickHouse support for self-hosted.

About Turso, I would say that if a remote database has to be used, currently supported PostgreSQL and MySQL are enough, not much can be gained here by using SQLite remotely IMO. But maybe I miss something.

And just a quick note that you can try Umami with SQLite here (https://github.com/Maxime-J/umami-sqlite).

**Comment by emilpriver** [comment]
I just think it's more about allowing more users to use whatever database they like and prefer and so on and turso works really good

**Comment by mikecao** [comment]
We have no plans to support more databases. It's not a matter of simply enabling another database through Prisma. You have to write custom logic for each database and support any issues that come up (looking at you MySQL).

**Reply by emilpriver** [reply]
Hey, I totally get this. My question was more if Turso was an option or not ☺️

**Comment by typedrat** [comment]
I'm also interested in Cloudflare D1 support. It's such an insane value that it's hard to consider using any other competing product.

**Reply by Root-FTW** [reply]
> I'm also interested in Cloudflare D1 support. It's such an insane value that it's hard to consider using any other competing product.

same here
